client/src/modules/dashboard/shared/components/Layout.tsx

```diff
@@ -17,17 +22,23 @@ export const Layout = ({ children }: { children: React.ReactNode }) => {
   return (
     <>
       <HStack as="nav" my="2">
-        {links.map((item) => (
-          <LinkButton
-            key={item.link}
-            href={item.link}
-            colorScheme={
-              router.pathname.startsWith(item.link) ? 'blue' : undefined
-            }
-          >
-            {item.text}
-          </LinkButton>
-        ))}
+        {links
+          .filter(
+            ({ permission }) =>
+              typeof permission === 'undefined' ||
+              useCheckPermission(permission),
+          )
+          .map((item) => (
+            <LinkButton
+              key={item.link}
+              href={item.link}
+              colorScheme={
+                router.pathname.startsWith(item.link) ? 'blue' : undefined
```

Comment:
```suggestion
                router.pathname.startsWith(item.link) ? 'blue' : 'gray'
```
Since we're here, `gray` is more declarative than `undefined`.